camelot/view/action_steps/form_view.py

```diff
@@ -154,6 +151,24 @@ def gui_run(cls, gui_context_name, serialized_step):
                 ))
                 show_top_level(formview, gui_context_name, step['form_state'])
 
+@dataclass
+class HighlightForm(ActionStep, DataclassSerializable):
+
+    tab: Optional[str] = None # The form tab
+    label: Optional[str] = None # A field label to highlight
+    label_delegate: bool = False # Highlight delegate associated with label
+    label_delegate_focus: bool = False # Focus delegate associated with label
+    table_label: Optional[str] = None # Label of the table for table_row and table_column
+    table_row: Optional[int] = None # Table row to highlight
+    table_column: Optional[str] = None # Table column to highlight
+    action_route: Optional[Route] = None # Action to highlight
+    action_menu_route: Optional[Route] = None # Menu to open
+    action_menu_mode: Optional[str] = None # Menu mode (verbose name) to highlight
+
+    #action_cls_state: Optional[?] = None
+    #group_box: Optional[?] = None
+    form_state: Optional[str] = None
+    field_name: Optional[str] = None
```

Comment:
Deze 4 moeten dus nog geimplementeerd worden...

Reply:
form_state hoort nt bij de highlight, deze hoort bij de OpenForm, die geeft aan of het form, full screen, left, right etc is